setup.py

```diff
@@ -35,14 +35,11 @@
         'dev': [
             "coverage==4.2",
             "flake8==3.2.1",
-            "mccabe==0.6.1",
-            "pycodestyle==2.3.1",
-            "pyflakes==1.5.0",
```

Comment:
I forgot that we pinned exact versions of dependencies used by tests. Will revert